transaction/core/tests/validation.rs

```diff
@@ -7,826 +7,842 @@ extern crate alloc;
 
 mod util;
 
-use alloc::vec::Vec;
-use mc_crypto_keys::{CompressedRistrettoPublic, ReprBytes};
-use mc_ledger_db::Ledger;
-use mc_transaction_core::{
-    constants::{MAX_TOMBSTONE_BLOCKS, RING_SIZE},
-    membership_proofs::Range,
-    tokens::Mob,
-    tx::{TxOutMembershipHash, TxOutMembershipProof},
-    validation::*,
-    BlockVersion, InputRules, Token,
-};
-use mc_transaction_core_test_utils::{InverseTxOutputsOrdering, INITIALIZE_LEDGER_AMOUNT};
-use mc_util_test_helper::get_seeded_rng;
-use util::*;
-
-#[test]
-// Should return MissingMemo when memos are missing in an output
-fn test_validate_memo_exists() {
-    let (tx, _) = create_test_tx(BlockVersion::ZERO);
-    let tx_out = tx.prefix.outputs.first().unwrap();
-
-    assert!(tx_out.e_memo.is_none());
-    assert_eq!(
-        validate_memo_exists(tx_out),
-        Err(TransactionValidationError::MissingMemo)
-    );
-
-    let (tx, _) = create_test_tx(BlockVersion::ONE);
-    let tx_out = tx.prefix.outputs.first().unwrap();
-
-    assert!(tx_out.e_memo.is_some());
-    assert_eq!(validate_memo_exists(tx_out), Ok(()));
-}
+mod tests {
```

Comment:
This is normally done for unit tests in a source file, but now, this is a separate tests module at crate level. There's no need to write mod tests here, this file is already entirely tests. So we should just not do this